doc/sphinx/addendum/rewrite-rules.rst

```diff
@@ -128,37 +129,35 @@ Universe polymorphic rules
 --------------------------
 
 Rewrite rules support universe and sort quality polymorphism.
-Universe levels and sort quality variables must be declared with the notation :n:`@{q1 q2|u1 u2+|+}`
-(the same notation as universe instance declarations);
-each variable must appear exactly once in the pattern.
-If any universe level isn't bound in the rule,
-as is often the case with the level of a pattern variable when it is a type,
-you need to make the universe instance extensible (with the final +).
-Universe level constraints, as inferred from the pattern, must imply those given,
-which in turn must imply the constraints needed for the replacement.
-You can make the declared constraints extensible
-so all inferred constraints from the left-hand side are used for the replacement.
+As with pattern variables, universe levels and sort quality variables
+must appear linearly in the pattern.
+Sort quality variables which appear only in relevances in the replacement
+will be detected if they also appear in a relevance in the pattern, such that
+they can be substituted when the rule is applied (otherwise you will get an undeclared sort quality error).
```

Comment:
What is a relevance?  I can't guess.  Are you maybe saying the sort variables in the replacement that are unbound in the pattern are not allowed?

Reply:
I added a link to the relevant manual entry